lib/ramble/ramble/application.py

```diff
@@ -89,6 +89,12 @@ class ApplicationBase(object, metaclass=ApplicationMeta):
     def __init__(self, file_path):
         super().__init__()
 
+        # Convert metaclass directives from class attributes to instance attributes
```

Comment:
Pretty nifty! Can this be made into a util function and shared with the modifier code?

Also curious if there's an example showing the usage of this change, or maybe a bug that's caused by the current behavior? Perhaps a test case would be informative.

Reply:
Done and done. :)